jekyll/_cci2/using-orbs.md

```diff
@@ -138,8 +136,10 @@ Before using orbs, you may find it helpful to understand the various design deci
 * One registry (per install) - Each installation of CircleCI, including circleci.com, has only one registry where orbs can be kept.
 * Organization Admins publish production orbs. Organization members publish development orbs - All namespaces are owned by an organization. Only the admin(s) of that organization can publish/promote a production orb. All organization members can publish development orbs.
 
-
 ## See Also
 {:.no_toc}
 
-[Reusing Config]({{site.baseurl}}/2.0/reusing-config/)
+- Refer to [Orb Introduction]({{site.baseurl}}/2.0/orb-intro/), for a high-level overview.
+- Refer to [Using Orbs]({{site.baseurl}}/2.0/using-orbs/), for more about how to use existing orbs.
```

Comment:
This is a link to the doc you are already in, remove this one

Reply:
Oops....yeah, forgot to remove this link. Fixed.